### ISSUE TITLE
Firefox 52 tooltip is left aligned and cut in screen

### ISSUE THREAD

**Issue by andrericardo** [danielpalme/ReportGenerator#72]
In Firefox 52.0 the on hover tooltip is miss aligned to the left in the screen and half of it not visible. Sorry but I cannot upload files with a screenshot.

Generated code in Firefox (half of the tooltip is off screen to left and not aligned with the data point)
`<div class="tooltip" style="left: -99px; top: -14px; display: none;">...</div>`

Same code in Chrome working fine
`<div class="tooltip" style="left: 389.5px; top: -17px; display: none;">...</div>`

I think the code causing it is this line
[https://github.com/danielpalme/ReportGenerator/blob/ba82de54cc1ab8fb055de4a9404d132d155458c0/ReportGenerator.Reporting/Rendering/resources/customAngularApp.js#L135](https://github.com/danielpalme/ReportGenerator/blob/ba82de54cc1ab8fb055de4a9404d132d155458c0/ReportGenerator.Reporting/Rendering/resources/customAngularApp.js#L135)

In Firefox 52 seems that values of offsetX are very small compared to layerX. 
Added some console.log lines
```
console.log("offsetX: " + event.offsetX);
console.log("layerX: " + event.originalEvent.layerX);
offsetX: 6    customAngularApp.js:138:5
layerX: 544   customAngularApp.js:139:5
```

Removing "event.offsetX ||" makes it work correctly for Firefox 52. At the moment seems that offsetX is in experimental mode in Firefox https://developer.mozilla.org/en-US/docs/Web/API/MouseEvent/offsetX#Browser_compatibility

Works for Firefox 52 and Chrome 37, not sure if would break other browsers.
```
left: (event.originalEvent.layerX) - tooltip.width() / 2 - 5,
top: (event.originalEvent.layerY) - tooltip.height() - 40
```

Thank you for making such a nice report generator.

**Comment by danielpalme** [comment]
Thanks for your detailed issue.
I will have a look at the problem.

**Comment by danielpalme** [comment]
You could try this beta release:
https://www.nuget.org/packages/ReportGenerator/2.5.2-beta1

**Comment by andrericardo** [comment]
In the readme.txt the CHANGELOG still lists "2.5.1.0" for this fix did you mean "2.5.2.0" ?

Tested 2.5.2-beta1 and tooltip position works fine in 
IE 11.0.9600.18376
Firefox 52.0a1
Google 37.0.2062.124 m

Thank you Daniel!

**Comment by danielpalme** [comment]
Thanks for testing the beta.
I will update the version in the change log in the final release.
